test/test_ops.py

```diff
@@ -514,7 +514,9 @@ def _fn(t, *args, **kwargs):
 
         return _fn
 
-    def _check_helper(self, device, dtype, op, variant, check, *, check_forward_ad=False):
+    def _check_helper(self, device, dtype, op, variant, check, *, check_forward_ad=False, check_backward_ad=True,
+                      check_undefined_grad=True):
+        # NB: check_backward_ad does not affect graddgradcheck (always True)
```

Comment:
extra d here.

Reply:
Also should we make that an assert: you should only run gradgradcheck if check_backward_ad is True.

Reply:
Yeah, currently though the OpInfo flags are setup so that they are all the checks are True by default, and if for example `supports_autograd` is False, `supports_gradgrad` would still be True, but just silently ignored.
I think we can make these dependency slightly more explicit by setting `supports_gradgrad` to False automatically in the OpInfo constructor if `supports_autograd` is False

Reply:
I guess it would still be silently ignored, but the issue is that we don't want to have to add 10 fields to OpInfo every time we want to disable autograd